pkg/cmd/server/origin/master_config.go

```diff
@@ -790,6 +790,11 @@ func (c *MasterConfig) KubeClient() *kclient.Client {
 	return c.PrivilegedLoopbackKubernetesClient
 }
 
+// OpenShiftClient returns the openshift client object
+func (c *MasterConfig) OpenShiftClient() *osclient.Client {
```

Comment:
I'd rather have an `OAuthServerClients() (*osclient.Client, *kclient.Client)` function to clarify what the returned clients are to be used for